gpbft/participant.go

```diff
@@ -35,8 +47,14 @@ func (p *Participant) ID() ActorID {
 
 // Fetches the preferred EC chain for the instance and begins the GPBFT protocol.
 func (p *Participant) Start() error {
-	chain, power, beacon := p.host.GetCanonicalChain()
 	var err error
+	defer func() {
+		if r := recover(); r != nil {
+			err = &PanicError{Err: err}
```

Comment:
This doesn't do anything without naming the return value. This just sets a local which is then ignored as the function continues to return the default zero value.

Reply:
Agreed, I realised of that after a commit during the merge and pushed [another commit](https://github.com/filecoin-project/go-f3/blob/ranchalp/catch-panics/gpbft/participant.go), but I guess somebody (perhaps myself) clicked merge before that commit. Will fix that in a new PR now.

Reply:
Sorry for missing this in CR.